src/features/view-nft/NFTDetailsCard/ui/ActionContainer/ActionContainer.module.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../../../../../node_modules/@zero-tech/zui/styles/_borders.scss';
+@import '../../../../../../node_modules/@zero-tech/zui/styles/_colors.scss';
+@import '../../../../../../node_modules/@zero-tech/zui/styles/_typography.scss';
+
+.ActionContainer {
+	display: flex;
+	flex-direction: column;
+	padding: 1.5rem;
+	background: #18101e;
+	border-radius: $radius-primary;
+	max-width: 20rem;
+	gap: 1rem;
+
+	@media only screen and (min-width: 485px) {
+		max-width: unset;
+	}
+
+	@media only screen and (min-width: 878px) {
```

Comment:
These viewport-based queries will break when zOS team adds the right-sidekick. Should make a note of this and take them out later.